proto/private/pbconfigentry/config_entry.proto

```diff
@@ -809,8 +815,9 @@ message ResourceReference {
 message BoundAPIGateway {
   map<string, string> Meta = 1;
   repeated BoundAPIGatewayListener Listeners = 2;
+  uint64 Hash = 3;
   // mog: func-to=serviceRefsToStructs func-from=serviceRefFromStructs
-  map<string, ListOfResourceReference> Services = 3;
+  map<string, ListOfResourceReference> Services = 4;
 }
```

Comment:
Is changing the unique field number here for `Services` intentional. Something something backwards compatibility / protobuf best practices?

Reply:
it's definitely not intentional, I think my IDE is doing things by it self 😅! Great catch!